server/protocols/jmap-draft-integration-testing/jmap-draft-integration-testing-common/src/test/java/org/apache/james/jmap/draft/methods/integration/GetMessageListMethodTest.java

```diff
@@ -2319,6 +2320,24 @@ public void getMessageListShouldReturnTwoMessagesWhenCopiedAtOnceViaIMAP() throw
             .until(() -> twoMessagesFoundInMailbox(otherMailboxId));
     }
 
+    @Test
+    public void sendingAMessageThroughAnotherProtocolShouldBePresentInOutboxViaJmap() throws Exception {
+        MailboxId outboxMailboxId = mailboxProbe.createMailbox(MailboxConstants.USER_NAMESPACE, ALICE, "Outbox");
+        String messageBody = "We're all mad here.";
+        mailboxProbe.appendMessage(
```

Comment:
it's not a message sending, the test name is no consistent with the fact you put it through internal APIs